src/helpers/appBar.js

```diff
@@ -75,7 +75,7 @@ const useStyles = makeStyles((theme) => ({
     padding: theme.spacing(1, 1, 1, 1),
     paddingLeft: `calc(1em + ${theme.spacing(4)}px)`,
     transition: theme.transitions.create("width"),
-    width: "500px",
+    width: "325px",
```

Comment:
There is still some issue when width becomes less than 400px. Try 100% instead of 325px, it should work fine.

Reply:
Thanks for the review! The issue does not persist anymore by using 100%. Mind taking a look again?